internal/database/authorized_app.go

```diff
@@ -25,10 +25,20 @@ import (
 	pgx "github.com/jackc/pgx/v4"
 )
 
+type AuthorizedAppModel struct {
```

Comment:
I Would move these out to a sub-package first

that would allow for better naming, like

authorizedappdb.Model

Reply:
Thanks! I had a sync with jba about this. I also got further notes from xtof. 

- Renaming back to AuthorizedAppDB
- Subpackages as noted
- Possibly at the end moving common db to be internal only, but let's not get ahead of things.